### DISCUSSION TITLE
Changes to .env not reflecting in dbmate

### DISCUSSION THREAD

**Discussion by faulander** [amacneil/dbmate#532] [Support] [answered]
i have changed the structure of my code and moved everything from a src folder to the root folder.
i have adapted the .env file to reflect these changes.
dmbate still wants to create or adapt the db in the original folder, where is this saved?
![CleanShot 2024-03-22 at 12 52 34@2x](https://github.com/amacneil/dbmate/assets/38588197/01a4fb30-563f-4097-a1fd-c56a74cd6994)

**Comment by dossy** [comment]
What's the output of `echo $DBMATE_MIGRATIONS_DIR` before you execute `dbmate up`?

**Comment by faulander** [comment, marked as answer]
Found the problem. had to restart the shell, then it worked.
Ticket can be closed.

Thanks for the great library!

Dossy Shiobara ***@***.***> schrieb am Sa. 23. März 2024 um
21:24:
